.gitignore

```diff
@@ -234,3 +234,4 @@ _Pvt_Extensions
 
 # FAKE - F# Make
 .fake/
+VS
```

Comment:
No, I have in there old-school vs projects which I update by hands with links to source files.

I need them to use Resharper Test Runner for debugging.